AlphaWallet/Transfer/Views/TransitionButton.swift

```diff
@@ -18,11 +18,13 @@ public enum StopAnimationStyle {
     case shake
 }
 
+fileprivate typealias CFG = Configuration
```

Comment:
Let's not use this.

Reply:
As in use a different `typealias` name or don't use the`typealias`? Removing the `typealias` I get an error message when I try to use `Configuration.Color.Semantic.defaultButtonBackground`. 

```
Type 'UIButton.Configuration' has no member color.
```

Reply:
@hboon

Reply:
Changing to 
```
    var shrinkBackgroundColor: UIColor = AlphaWallet.Configuration.Color.Semantic.defaultButtonBackground
```
solved this issue. I believe there is another occurrence of this. I will put in another PR to correct the other one too.